### ISSUE TITLE
MICC error: there is no package called 'VGAM'

### ISSUE THREAD

**Issue by bioinfo89** [GuipengLi/ChIA-PET2#28]
Hi @GuipengLi ,

I get the following error message at step 7:

```
Loading required package: VGAM

Warning message:

In library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :

  there is no package called 'VGAM'

Error in zeta(theta) : could not find function "zeta"

Calls: MICCoutput2 ... optim -> <Anonymous> -> fn -> logF1 -> ZetaTruncated

Execution halted
```

I tried to install VGAM separately and exported its path, but it still gives this error and the script halts. I am using R 3.6.3.  It worked all right before not sure what changed now. Any help would be appreciated. Thanks!

**Comment by GuipengLi** [comment]
I tested it with R 3.6.3 and it works in my case. Are there more than one version of R installed ?

~$ R

R version 3.6.3 (2020-02-29) -- "Holding the Windsock"
Copyright (C) 2020 The R Foundation for Statistical Computing
Platform: x86_64-pc-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

  Natural language support but running in an English locale

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.

> install.packages("VGAM")
Installing package into ‘/home/lee/R/x86_64-pc-linux-gnu-library/3.6’
(as ‘lib’ is unspecified)
trying URL 'https://mirrors.tuna.tsinghua.edu.cn/CRAN/src/contrib/VGAM_1.1-5.tar.gz'
Content type 'application/octet-stream' length 1580498 bytes (1.5 MB)
==================================================
downloaded 1.5 MB

* installing *source* package ‘VGAM’ ...
** package ‘VGAM’ successfully unpacked and MD5 sums checked
** using staged installation
** libs
gcc -std=gnu99 -I"/usr/share/R/include" -DNDEBUG     -fpic  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c VGAM_init.c -o VGAM_init.o
gcc -std=gnu99 -I"/usr/share/R/include" -DNDEBUG     -fpic  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c caqo3.c -o caqo3.o
gfortran  -fpic  -g -O2 -fstack-protector-strong  -c ei.f -o ei.o
gfortran  -fpic  -g -O2 -fstack-protector-strong  -c fgam.f -o fgam.o
gcc -std=gnu99 -I"/usr/share/R/include" -DNDEBUG     -fpic  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c gautr.c -o gautr.o
gcc -std=gnu99 -I"/usr/share/R/include" -DNDEBUG     -fpic  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c lerchphi.c -o lerchphi.o
gfortran  -fpic  -g -O2 -fstack-protector-strong  -c lms.f -o lms.o
gcc -std=gnu99 -I"/usr/share/R/include" -DNDEBUG     -fpic  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c muxr3.c -o muxr3.o
gfortran  -fpic  -g -O2 -fstack-protector-strong  -c rgam.f -o rgam.o
gcc -std=gnu99 -I"/usr/share/R/include" -DNDEBUG     -fpic  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c rgam3.c -o rgam3.o
gcc -std=gnu99 -I"/usr/share/R/include" -DNDEBUG     -fpic  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c specfun3.c -o specfun3.o
gfortran  -fpic  -g -O2 -fstack-protector-strong  -c tyeepolygamma.f -o tyeepolygamma.o
gcc -std=gnu99 -I"/usr/share/R/include" -DNDEBUG     -fpic  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c tyeepolygamma3.c -o tyeepolygamma3.o
gfortran  -fpic  -g -O2 -fstack-protector-strong  -c vdigami.f -o vdigami.o
gfortran  -fpic  -g -O2 -fstack-protector-strong  -c veigen.f -o veigen.o
gfortran  -fpic  -g -O2 -fstack-protector-strong  -c vgam.f -o vgam.o
gcc -std=gnu99 -I"/usr/share/R/include" -DNDEBUG     -fpic  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c vgam3.c -o vgam3.o
gfortran  -fpic  -g -O2 -fstack-protector-strong  -c vlinpack1.f -o vlinpack1.o
gfortran  -fpic  -g -O2 -fstack-protector-strong  -c vlinpack2.f -o vlinpack2.o
gfortran  -fpic  -g -O2 -fstack-protector-strong  -c vlinpack3.f -o vlinpack3.o
gfortran  -fpic  -g -O2 -fstack-protector-strong  -c vmux.f -o vmux.o
gcc -std=gnu99 -I"/usr/share/R/include" -DNDEBUG     -fpic  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c vmux3.c -o vmux3.o
gcc -std=gnu99 -I"/usr/share/R/include" -DNDEBUG     -fpic  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c zeta3.c -o zeta3.o
gcc -std=gnu99 -shared -L/usr/lib/R/lib -Wl,-Bsymbolic-functions -Wl,-z,relro -o VGAM.so VGAM_init.o caqo3.o ei.o fgam.o gautr.o lerchphi.o lms.o muxr3.o rgam.o rgam3.o specfun3.o tyeepolygamma.o tyeepolygamma3.o vdigami.o veigen.o vgam.o vgam3.o vlinpack1.o vlinpack2.o vlinpack3.o vmux.o vmux3.o zeta3.o -lgfortran -lm -lquadmath -L/usr/lib/R/lib -lR
installing to /home/lee/R/x86_64-pc-linux-gnu-library/3.6/00LOCK-VGAM/00new/VGAM/libs
** R
** data
*** moving datasets to lazyload DB
** demo
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (VGAM)

The downloaded source packages are in
	‘/tmp/RtmphzDuKs/downloaded_packages’

> library(VGAM)
Loading required package: stats4
Loading required package: splines

**Comment by bioinfo89** [comment]
Yes there are more than one version of R installed on the server. But I am using module load R/3.6.3.

I followed the exact steps to install VGAM using R/3.6.3 but I get the error when I run ChIAPET from step 7.

Best,
Khyati Chandratre

**Comment by GuipengLi** [comment]
Please check which Rscript you are using.

**Comment by bioinfo89** [comment]
I checked it is 3.6.3. I am not sure the tool is still not able to find the module.
